azavea/franklin
# Add tile rendering support

## Overview

This adds tile rendering support for COG assets

### Checklist


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

### Testing Instructions

- Import a catalog from S3: `bloop run application -- import --catalog-root s3://rasterfoundry-development-data-us-east-1/berlin-catalog/catalog.json --db-host localhost`
- Start server with tiles: `./scripts/server --with-tiles`
- Go to `http://localhost:9090/collections/berlin/items/image` -- verify that there is a tiles link
- Click on the tiles link and copy/paste the URL (substituting `z/x/y` for the matrix, col, row parameters into something that makes tile requests, verify it works

Closes #146
